contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -0,0 +1,1186 @@
+     * than or equal to the value of project `_project_id`'s maximum invocations that is
+     * set on the core contract.
+     * @dev Note that a `_maxInvocations` of 0 can only be set if the current `invocations`
+     * value is also 0 and this would also set `maxHasBeenInvoked` to true, correctly short-circuiting
+     * this minter's purchase function, avoiding extra gas costs from the core contract's maxInvocations check.
+     * @param _projectId Project ID to set the maximum invocations for.
+     * @param _maxInvocations Maximum invocations to set for the project.
+     */
+    function manuallyLimitProjectMaxInvocations(
+        uint256 _projectId,
+        uint256 _maxInvocations
+    ) external {
+        _onlyArtist(_projectId);
+        // CHECKS
+        // ensure that the manually set maxInvocations is not greater than what is set on the core contract
+        uint256 maxInvocations;
+        uint256 invocations;
+        (invocations, maxInvocations, , , , ) = genArtCoreContract_Base
+            .projectStateData(_projectId);
+        require(
+            _maxInvocations <= maxInvocations,
+            "Cannot increase project max invocations above core contract set project max invocations"
+        );
+        require(
+            _maxInvocations >= invocations,
+            "Cannot set project max invocations to less than current invocations"
+        );
+        // EFFECTS
+        // update storage with results
+        projectConfig[_projectId].maxInvocations = uint24(_maxInvocations);
+        // We need to ensure maxHasBeenInvoked is correctly set after manually
+        // setting the local maxInvocations value.
+        projectConfig[_projectId].maxHasBeenInvoked =
+            invocations == _maxInvocations;
+
+        emit ProjectMaxInvocationsLimitUpdated(_projectId, _maxInvocations);
+
+        // for convenience, try to mint and assign a token to the project's
+        // next slot
+        _tryMintTokenToNextSlot(_projectId);
+    }
+
+    /**
+     * @notice Sets the minimum and maximum values that are settable for
+     * `durationSeconds` for all project configurations.
+     * Note that the auction duration is the initial duration of the auction,
+     * and does not include any extensions that may occur due to new bids being
+     * placed near the end of an auction.
+     * @param _minAuctionDurationSeconds Minimum auction duration in seconds.
+     * @param _maxAuctionDurationSeconds Maximum auction duration in seconds.
+     */
+    function updateAllowableAuctionDurationSeconds(
+        uint32 _minAuctionDurationSeconds,
+        uint32 _maxAuctionDurationSeconds
+    ) external {
+        _onlyCoreAdminACL(this.updateAllowableAuctionDurationSeconds.selector);
+        // CHECKS
+        require(
+            _maxAuctionDurationSeconds > _minAuctionDurationSeconds,
+            "Only max gt min"
+        );
+        require(_minAuctionDurationSeconds > 0, "Only min gt 0");
```

Comment:
Nit: we do a lot of >0 checks, wonder if we should make them into a helper `_onlyNonZero` method?

Reply:
I like this idea. I broke it out into two functions since I think we kind of have two types of common non-zero tests in this minter:
- a modifier-like `_onlyNonZero` function that may be used to require function inputs be non-zero
- an informative `_projectIsConfigured` internal method, that checks if a project is configured via checking `projectConfig.basePrice`
These updates were pushed in commit [12dfb02](https://github.com/ArtBlocks/artblocks-contracts/pull/504/commits/12dfb0208541f9a40f3483d1b4ae727f8860c20f)

Reply:
Awesome – thank you! LGTM.